opm/autodiff/StandardWell.hpp

```diff
@@ -28,6 +28,8 @@
 #include <opm/autodiff/ISTLSolverEbos.hpp>
 #include <opm/autodiff/RateConverter.hpp>
 
+#include <string>
```

Comment:
most likely, we do not need this `include` here.

Reply:
As std::to_string is used in the file, it is correct to include it.

Reply:
but std::string is everywhere, so I am not sure about it. I can be wrong.

Reply:
I removed it and it works fine..:)